bigquery_etl/cli/query.py

```diff
@@ -0,0 +1,144 @@
+"""bigquery-etl CLI query command."""
+
+import click
+import os
+from pathlib import Path
+import re
+import sys
+import string
+
+from bigquery_etl.metadata.parse_metadata import Metadata
+
+
+QUERY_NAME_RE = re.compile(
+    r"(?P<dataset>[a-zA-z0-9_]+)\.(?P<name>[a-zA-z0-9_]+)(?P<ver>_v[0-9]+)?"
+)
+
+
+def create_query_cli_group():
+    """Create the CLI group for the query command."""
+    group = click.Group(name="query", help="Creating/validating/deleting/... queries")
+    group.add_command(query_create_command)
+    return group
+
+
+@click.command(
+    name="create",
+    help=(
+        "Create a new query with name "
+        "<dataset>.<query_name>, for example: telemetry_derived.asn_aggregates"
+    ),
+)
+@click.argument("name")
+@click.option(
+    "--path",
+    "-p",
+    help="Path to directory in which query should be created",
+    default="sql/",
```

Comment:
You can [specify path validation](https://click.palletsprojects.com/en/7.x/api/#click.Path): 
```suggestion
    type=click.Path(file_okay=False),
    default="sql/",
```